integration_test/ops_agent_test/main_test.go

```diff
@@ -206,6 +206,26 @@ func retrieveOtelConfig(ctx context.Context, logger *log.Logger, vm *gce.VM) (co
 	return gce.RetrieveContent(ctx, logger, vm, agents.GetOtelConfigPath(vm.ImageSpec))
 }
 
+// RunForEachLoggingSubagent runs a subtest for the logging subagent fluent-bit and otel.
+func RunForEachLoggingSubagent(t *testing.T, testBody func(t *testing.T, otel bool)) {
+	t.Run("fluent-bit", func(t *testing.T) {
+		testBody(t, false)
+	})
+
+	t.Run("otel", func(t *testing.T) {
+		testBody(t, true)
+	})
+}
+
+// setOtelLoggingEnvironmentVariable sets the EXPERIMENTAL_FEATURES: otel_logging environment variable.
+func setOtelLoggingEnvironmentVariable(ctx context.Context, logger *log.Logger, vm *gce.VM, otel bool) error {
```

Comment:
Call it `setExperimentalFeatures` and have it take a string so it's not so specific?

Reply:
Yeah! I agree! It was too specific. Done!